recipes/perl-vcftools-vcf/840/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -eu
```

Comment:
Should probably be `set -euo pipefail` for consistency (even though there are no pipes, others might use this as a template for their own build script)